owncloud/ios-app
# [fix/upload-path-visibility] Fix for Upload Path in Settings

## Description
Photo/video upload path was not visible in some cases (when it was disabled before and path already exists in user defaults)

## Related Issue
#784 

## Motivation and Context
Give the user UI feedback after enabling the upload path option

## How Has This Been Tested?
- enable photo or video upload path
- select account
- select path
- close settings
- open settings again
- disable photo or video upload path
- close settings
- open settings again
- enable photo or video upload path

Before the fix, the path will not be shown in UI, after the fix the path will appear

## Types of changes


## Checklist:
